src/views/GovernanceTaskList/index.tsx

```diff
@@ -159,6 +164,13 @@ const GovernanceTaskList = () => {
     }
   }, [dispatch, systemIntake.id, systemIntake.businessCaseId]);
 
+  useEffect(() => {
```

Comment:
This automatically expands the remaining steps if the status reflects the requester is on one of those steps. We will need to incrementally add statuses to `remainingStepsStatuses`.